src/main/java/org/cascadebot/cascadebot/commands/informational/UserInfoCommand.java

```diff
@@ -77,7 +77,7 @@ public void onCommand(Member sender, CommandContext context) {
         }
 
         if (member != null) {
-            builder.addField("Status", status + statusName, true);
+            builder.addField(context.i18n("commands.userinfo.status"), status + statusName, true);
```

Comment:
I have a feeling this was in the words section

Reply:
Yeah it is `words.status`, can you delete `commands.userinfo.status` from the lang and change it where it's used